lora-phy/src/sx1261_2/mod.rs

```diff
@@ -120,28 +120,26 @@ where
     // Set the number of symbols the radio will wait to detect a reception
     async fn set_lora_symbol_num_timeout(&mut self, symbol_num: u16) -> Result<(), RadioError> {
         let mut exp = 0u8;
-        let mut reg;
-        let mut mant = ((core::cmp::min(symbol_num, SX126X_MAX_LORA_SYMB_NUM_TIMEOUT as u16) as u8) + 1) >> 1;
+        let mut mant: u8 = (symbol_num as u8).min(SX126X_MAX_LORA_SYMB_NUM_TIMEOUT);
```

Comment:
```suggestion
        let mut mant: u8 = (symbol_num).min(SX126X_MAX_LORA_SYMB_NUM_TIMEOUT) as u8;
```
I think we need to worry about `256 as u8 = 1`?